### PR TITLE
Implement dable ad

### PR DESCRIPTION
**Pull request by mctenshi**
Provides support for amp-embed type=dable

For more information about Dable, please refer to http://dable.io/en/

### CODE REVIEW

**Comment by mctenshi** [comment]
I signed it!

File: examples/ads.amp.html [hunk 1]
```diff
@@ -582,6 +583,13 @@ <h2 id="custom">Custom leaderboard</h2>
       data-eas-cu="3255">
   </amp-ad>
 
+  <h2>Dable</h2>
+  <amp-embed width="450" height="140"
+      type="dable"
+      data-widget_id="jobgqR7W"
```
Comment (lannka) [inline examples/ads.amp.html]: format `data-widget-id` is preferred. you can read it from data object using the camel format:

`data.widgetId`

File: ads/dable.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (lannka) [inline ads/dable.js]: 2017

**Comment by mctenshi** [comment]
Fixed review issues.

**Comment by lannka** [comment]
@mctenshi  you probably missed my last comment:

> could you implement renderStart API?

**Comment by mctenshi** [comment]
@lannka Added `renderStartImplemented` flag on `ads/_config.js`. Logic that call `renderStart()` has already implemented on `ads/dable.js`. Thanks.

**Comment by mctenshi** [comment]
@lannka Could I use different email address from one that I used in first commit?

I force updated first commit with new email address, and used new email address in 2nd/3rd commit but cla bot seems not recognize the change.

**Comment by lannka** [comment]
@mctenshi I'm contacting the maintainer of the CLA robot to see what they say.

**Comment by lannka** [comment]
@mctenshi can you double check if you've signed CLA using this email: `tenshi@dable.io`?

**Comment by mctenshi** [comment]
> can you double check if you've signed CLA using this email: tenshi@dable.io?

@lannka Yes

**Comment by lannka** [comment]
This is what you should see on https://cla.developers.google.com/ . please confirm the email address on the top right corner

screen shot 2017-07-20 at 6 47 59 pm

**Comment by mctenshi** [comment]
@lannka Thank you, I got new CLA. I did it via "Your Employer" button but maybe it did not make it.

**Pull request merged**